quickcheck-state-machine.cabal

```diff
@@ -23,7 +23,7 @@ extra-doc-files:
   README.md
 
 tested-with:
-  GHC ==8.8.4 || ==8.10.7 || ==9.2.8 || ==9.4.7 || ==9.6.3 || ==9.8.1
```

Comment:
`tested-with` should correspond to this list: https://github.com/stevana/quickcheck-state-machine/blob/master/.github/workflows/main.yaml#L23 , please keep exact versions and add 9.10.1 to CI.

Reply:
Done